### PR TITLE
Feature/547 tests for Library folder

### PR DESCRIPTION
**Pull request by lightme-fan**
## Tests in the `/lib` folder
- [x] api.test.js
- [x] geo.test.js
- [x] store.test.js
- [x] queue.test.js
- [x] ui-text.test.js

Screenshot 2022-07-06 17:02:45

### CODE REVIEW

File: frontend/src/lib/geo.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const tile = {
   attribution: "Tiles &copy; Esri &mdash; DeLorme, NAVTEQ, Esri",
 };
 
-const getBounds = (administration) => {
+export const getBounds = (administration) => {
```
Comment (dedenbangkit) [inline frontend/src/lib/geo.js:13]: Why do you need to export this? getBounds is already part of geo.

geo.getBounds()

File: frontend/src/lib/geo.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ const getBounds = (administration) => {
   };
 };
 
-const defaultPos = () => {
+export const defaultPos = () => {
```
Comment (dedenbangkit) [inline frontend/src/lib/geo.js:44]: same as above

File: frontend/src/pages/login/__test__/Login.test.js [hunk 3]
```diff
@@ -1,45 +0,0 @@
-import { render, screen } from "@testing-library/react";
```
Comment (dedenbangkit) [inline frontend/src/pages/login/__test__/Login.test.js:1]: Why you remove this?

File: frontend/src/pages/login/__test__/__snapshots__/Login.test.js.snap [hunk 4]
```diff
@@ -1,681 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (dedenbangkit) [inline frontend/src/pages/login/__test__/__snapshots__/Login.test.js.snap:1]: And also we need a snapshot, this is the example test I made

**Review by dedenbangkit — CHANGES_REQUESTED** [review]
Please check the comment @lightme-fan

**Comment by lightme-fan** [comment]
Thanks for the comments. Fixing them now.

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**